Adding Translation in Brazilian Portuguese

KoreaComK opened the pull request:
Hi,

I finished the translation to Brazilian Portuguese of the app using the transifex.

Cheers,

theborakompanioni commented:
Nice @KoreaComK , what a wonderful service to all Portuguese speaking people :pray: 

Would it be possible to delete one of the two translation files? Just keep either `pt_BR` or `ptbr` - I'd opt for keeping the `pt_BR` one, if you are okay with it. Also, please run `npm run format` to make the linter happy. After that, happy to merge :tada:

KoreaComK commented:
@theborakompanioni thanks for helping.
I run the `npm run format` and it changes `package-lock.json` and `package.json`. I didn't put both in this commit, since it just change the version of `prettier`.
Also I deleted the folder and file `ptbr` as mention, and changed the imports on `language.js` .
If it needs anything else, please, let me know.

Cheers!

theborakompanioni reviewed (approved):
tACK. :portugal: :brazil:
